src/vs/workbench/contrib/accessibility/browser/accessibleView.ts

```diff
@@ -774,10 +774,9 @@ function shouldHide(event: KeyboardEvent, keybindingService: IKeybindingService)
 	if (keybindingService.inChordMode || isValidChord) {
 		return false;
 	}
-	return isLetter(event.keyCode) && !event.ctrlKey && !event.altKey && !event.metaKey && !event.shiftKey;
+	return shouldHandleKey(event) && !event.ctrlKey && !event.altKey && !event.metaKey && !event.shiftKey;
 }
 
-function isLetter(code: number): boolean {
-	const key = String.fromCharCode(code);
-	return /^\p{L}$/u.test(key);
+function shouldHandleKey(event: KeyboardEvent): boolean {
+	return event.code.match(/^Key{A-Z}|Digit{0-9}|Equal|Comma|Period|Slash|Quote|Backquote|Backslash|Minus|Semicolon|Space$/) !== undefined;
```

Comment:
It actually returns null:

```suggestion
	return !!event.code.match(/^Key{A-Z}|Digit{0-9}|Equal|Comma|Period|Slash|Quote|Backquote|Backslash|Minus|Semicolon|Space$/);
```